CLAUDE.md

```diff
@@ -0,0 +1,40 @@
+# US Imputation Benchmarking - Developer Guide
+
+## Build & Test Commands
+```bash
+# Install package in development mode
+pip install -e .
+
+# Run all tests
+python us_imputation_benchmarking/tests.py
```

Comment:
Can we move these to a makefile? See the policyengine-uk or -us repo for an example.